### PR TITLE
Fixed datagrid mass action issue

### PR DESCRIPTION
**Pull request by prabhatkumaroc082**
**Describe the issue**
Datagrid mass update feature is not working due to object used [here](https://github.com/bagisto/bagisto/blob/master/packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/partials/datagrid-table.vue#L57)

**Expected result**
datagrid mass update must function

### CODE REVIEW

**Comment by devansh-webkul** [comment]
Need to check these cases also,
https://github.com/bagisto/bagisto/blob/master/packages/Webkul/Category/src/Http/Controllers/CategoryController.php#L217

**Comment by Sanjaybhattwebkul** [comment]
@prabhatkumaroc082 
The **mass** action is **working** in the admin panel, can you please **elaborate**?

**Comment by prabhatkumaroc082** [comment]
> Need to check these cases also, https://github.com/bagisto/bagisto/blob/master/packages/Webkul/Category/src/Http/Controllers/CategoryController.php#L217

@devansh-webkul applied changes.

File: packages/Webkul/Category/src/Http/Controllers/CategoryController.php [hunk 1]
```diff
@@ -214,7 +214,10 @@ public function massUpdate()
     {
         $data = request()->all();
 
-        if (! isset($data['mass-action-type']) || ! $data['mass-action-type'] == 'update') {
+        if (
+            !isset($data['mass-action-type'])
```
Comment (devansh-webkul) [inline packages/Webkul/Category/src/Http/Controllers/CategoryController.php:218]: Add space after `!`.

File: packages/Webkul/Product/src/Http/Controllers/ProductController.php [hunk 2]
```diff
@@ -300,8 +300,9 @@ public function massUpdate()
     {
         $data = request()->all();
 
-        if (! isset($data['mass-action-type']) 
-            || ! $data['mass-action-type'] == 'update'
+        if (
+            !isset($data['mass-action-type'])
```
Comment (devansh-webkul) [inline packages/Webkul/Product/src/Http/Controllers/ProductController.php:304]: Same here.

File: packages/Webkul/Product/src/Http/Controllers/ReviewController.php [hunk 3]
```diff
@@ -156,11 +156,10 @@ public function massUpdate()
                 $review = $this->productReviewRepository->find($value);
 
                 try {
-                    if (! isset($data['mass-action-type'])) {
-                        return redirect()->back();
-                    }
-
-                    if (! $data['mass-action-type'] == 'update') {
+                    if (
+                        !isset($data['mass-action-type'])
```
Comment (devansh-webkul) [inline packages/Webkul/Product/src/Http/Controllers/ReviewController.php:160]: Same here.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by devansh-webkul** [comment]
@Sanjaybhattwebkul,

Please test this.

**Comment by prabhatkumaroc082** [comment]
> @prabhatkumaroc082 The **mass** action is **working** in the admin panel, can you please **elaborate**?

@Sanjaybhattwebkul 

you will find object [here](https://user-images.githubusercontent.com/50438561/202662517-af8f4ad5-1d05-4cfb-8161-f335318bee57.png) in dropdown values.

you might not be facing the issue due to validation added [here](https://github.com/bagisto/bagisto/blob/master/packages/Webkul/Category/src/Http/Controllers/CategoryController.php#L217) if you closely look into condition `! $data['mass-action-type'] == 'update'` it will always return false even if key 'mass-action-type' doesn't contain value 'update'

i applied some improvements so that these conditions should work only when mass type is 'update'. For now, this validation is not in use.

**Comment by Sanjaybhattwebkul** [comment]
@prabhatkumaroc082 
Please resolve the conflicts.

**Comment by prabhatkumaroc082** [comment]
@Sanjaybhattwebkul review now.

**Comment by Sanjaybhattwebkul** [comment]
@devansh-webkul 
The **mass** actions are **working fine** in admin panel ,please find the screenshot [Here](https://prnt.sc/ojlXCNslnKCW)

**Review by devansh-webkul — APPROVED** [review]

**Pull request merged**